src/components/AudioTrack/layouts/Row.vue

```diff
@@ -29,15 +29,17 @@
               <template #creator>{{ audio.creator }}</template>
             </i18n>
             <span v-if="!isSmall">
-              • {{ timeFmt(audio.duration) }} •
+              {{ seperator }} {{ timeFmt(audio.duration) }}
```

Comment:
Sorry for not noticing this earlier, 'separator' is spelled incorrectly.
```suggestion
              {{ separator }} {{ timeFmt(audio.duration) }}
```

Reply:
ha, thanks for noticing.